dockerfiles/Dockerfile-toolchain

```diff
@@ -71,51 +71,54 @@ RUN sudo git clone https://github.com/facebook/rocksdb -b v5.17.2 /rocksdb
 # This builds and installs just the static lib for us
 RUN cd /rocksdb && sudo make static_lib PORTABLE=1 -j$(nproc) && sudo cp librocksdb.a /usr/local/lib/librocksdb_coda.a && sudo rm -rf /rocksdb && sudo strip -S /usr/local/lib/librocksdb_coda.a
 
-# clone Coda repo to pin external package and save their commits
-RUN sudo git clone https://github.com/CodaProtocol/coda $CODA_DIR
-
-# use HTTPS git clone for submodules
-RUN cd $CODA_DIR && sudo sed --in-place s+git@github.com:+https://github.com/+g .gitmodules
-
-# init submodules for pinning
-RUN cd $CODA_DIR && sudo git submodule update --init --recursive
+## clone Coda repo for the .nix-keys.gpg.asc
 
-# save submodule commits
-RUN for pkg in async_kernel digestif graphql_ppx ocaml-extlib rpc_parallel ; do \
-      # remove leading "-" indicating uninitialized
-      cd $CODA_DIR && sudo bash -c "git submodule status src/external/$pkg | awk '{print \$1}' | sed s/-// > ~opam/opam-repository/$pkg.commit" ; \
-    done
-
-# save log commits for ocaml-sodium, coda_base58, which are not submodules
-RUN for pkg in ocaml-sodium coda_base58 ; do \
-      cd $CODA_DIR && sudo bash -c "git log --format=oneline -n 1 src/external/$pkg | awk '{print \$1}' > ~opam/opam-repository/$pkg.commit" ; \
-    done
+RUN sudo git clone https://github.com/CodaProtocol/coda $CODA_DIR
```

Comment:
I think this step was to get the submodule pinning. If the pinning is removed, is this needed?

Reply:
yes, we get the gpg keys from this repo